Cannot find ./util/util.js

Before getting to #8, I originally got a
```
Error: Cannot find module './util/util.js'
```
This was solved by changing line 47 of `dist/index.js` from
```javascript
var accountPair = require('./util/util.js').accountPair;
```
to
```javascript
var accountPair = require('../util/util.js').accountPair;
```

--- a/README.md
+++ b/README.md
@@ -111,7 +111,8 @@ Used for generating accounts and extrapolating public keys/addresses from privat
 Account methods take a single account string or in some cases, an array of accounts.
 
 * `nano.accounts.get(publicKey: string)`
-* `nano.accounts.balance(account: string)`
+* `nano.accounts.rawBalance(account: string)`
+* `nano.accounts.nanoBalance(account: string)`
 * `nano.accounts.balances(accounts: string[])`
 * `nano.accounts.blockCount(account: string)`
 * `nano.accounts.frontiers(accounts: string[])`

--- a/index.ts
+++ b/index.ts
@@ -45,7 +45,7 @@ export interface NanoConstructorOptions {
   debug?: boolean
 }
 
-export default class Nano {
+class Nano {
   rpc = createAPI<API>(null)
   debug: boolean
 
@@ -535,3 +535,7 @@ export default class Nano {
     }
   }
 }
+
+// Support ES6 default import and CommonJS for node
+export default Nano
+module.exports = Nano

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   "scripts": {
     "debug": "ts-node --inspect-brk=9229 --debug examples/accounts.ts",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "prepublish": "tsc"
+    "prepublish": "tsc && copyfiles util/*.js dist"
   },
   "repository": {
     "type": "git",
@@ -20,19 +20,18 @@
   },
   "homepage": "https://github.com/mikepaszkiewicz/nanode#readme",
   "dependencies": {
-    "@types/bignumber.js": "5.0.0",
-    "@types/node": "^9.4.0",
     "axios": "^0.16.2",
-    "bignumber.js": "6.0.0",
-    "dotenv": "^5.0.0",
-    "moment": "^2.20.1",
-    "typescript": "^2.4.0"
+    "bignumber.js": "6.0.0"
   },
   "devDependencies": {
+    "@types/node": "9.4.6",
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
     "babel-polyfill": "^6.26.0",
     "babel-preset-es2015": "^6.24.1",
-    "babel-preset-stage-0": "^6.24.1"
+    "babel-preset-stage-0": "^6.24.1",
+    "copyfiles": "1.2.0",
+    "dotenv": "5.0.0",
+    "typescript": "2.7.2"
   }
 }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "nanode",
-  "version": "2.0.0",
+  "version": "2.1.0",
   "description": "Dead simple, promise-based client for interacting and building services on top of the NANO network.",
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",

